Exiled.Events/Patches/Events/Player/ActivatingWarheadPanel.cs

```diff
@@ -7,72 +7,151 @@
+                new CodeInstruction(OpCodes.Ldstr, permission),
+                new CodeInstruction(OpCodes.Call, Method(typeof(List<string>), nameof(List<string>.Add))),
+
+                // Player.Get(this.gameObject)
+                new CodeInstruction(OpCodes.Ldarg_0),
+                new CodeInstruction(OpCodes.Call, PropertyGetter(typeof(Component), nameof(Component.gameObject))),
+                new CodeInstruction(OpCodes.Call, Method(typeof(Player), nameof(Player.Get), new[] { typeof(GameObject) })),
+
+                // list
+                new CodeInstruction(OpCodes.Ldloc_S, list.LocalIndex),
+
+                // if (this._sr.BypassMode)
+                //   goto checkedBypassLabel;
+                new CodeInstruction(OpCodes.Ldarg_0),
+                new CodeInstruction(OpCodes.Ldfld, Field(typeof(PlayerInteract), nameof(PlayerInteract._sr))),
+                new CodeInstruction(OpCodes.Ldfld, Field(typeof(ServerRoles), nameof(ServerRoles.BypassMode))),
+                new CodeInstruction(OpCodes.Brtrue_S, checkedBypassLabel),
+
+                // itemByID.permissions.Contains("CONT_LVL_3")
+                //   goto checkedContainsLabel
+                new CodeInstruction(OpCodes.Ldloc_1),
+                new CodeInstruction(OpCodes.Ldfld, Field(typeof(global::Item), nameof(global::Item.permissions))),
+                new CodeInstruction(OpCodes.Ldstr, permission),
+                new CodeInstruction(OpCodes.Call, typeof(Enumerable).GetMethods().FirstOrDefault(method => method.Name == "Contains" && method.GetParameters().Length == 2).MakeGenericMethod(typeof(string))),
+                new CodeInstruction(OpCodes.Br_S, checkedContainsLabel),
+
+                new CodeInstruction(OpCodes.Ldc_I4_1).WithLabels(checkedBypassLabel),
+
+                // var ev = new ActivatingWarheadPanelEventArgs(...)
+                new CodeInstruction(OpCodes.Newobj, GetDeclaredConstructors(typeof(ActivatingWarheadPanelEventArgs))[0]).WithLabels(checkedContainsLabel),
+                new CodeInstruction(OpCodes.Dup),
+
+                // Handlers.Player.OnActivatingWarheadPanel(ev);
+                new CodeInstruction(OpCodes.Call, Method(typeof(Handlers.Player), nameof(Handlers.Player.OnActivatingWarheadPanel))),
+
+                // ListPool<string>.Shared.Return(list);
+                new CodeInstruction(OpCodes.Ldsfld, Field(typeof(ListPool<string>), nameof(ListPool<string>.Shared))),
+                new CodeInstruction(OpCodes.Ldloc_S, list.LocalIndex),
+                new CodeInstruction(OpCodes.Callvirt, Method(typeof(ListPool<string>), nameof(ListPool<string>.Return))),
+
+                // if (ev.IsAllowed)
+                //   goto insideIfLabel
+                new CodeInstruction(OpCodes.Callvirt, PropertyGetter(typeof(ActivatingWarheadPanelEventArgs), nameof(ActivatingWarheadPanelEventArgs.IsAllowed))),
+                new CodeInstruction(OpCodes.Brtrue_S, insideIfLabel),
```

Comment:
Use `brtrue` instead

Reply:
Why? `insideIfLabel` is like 2 instructions below, there is no need to use the not short form.